content/hi/partners/_index.html

```diff
@@ -0,0 +1,88 @@
+---
+title: सहभागी
+bigheader: कूबरनेट्स सहभागी
+abstract: कूबरनेट्स इकोसिस्टम का विस्तार.
+class: gridPage
+cid: सहभागी
+---
+
+<section id="users">
+    <main>
+        <h5>कुबरनेट्स सहयोगियों संग एक मज़बूत कोड बेस का निर्माण करता है जो संपूरक मंचों के स्पेक्ट्रम को सहयोग करता है।</h5>
+        <div class="col-container">
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>कुबरनेट्स प्रमाणित सेवा प्रदाता</b>
+              </h5>
+              <br>उद्यमों द्वारा कुबरनेट्स सफलतापूर्वक अपनाने में सहभागी अनुभवी सेवा प्रदाता।
```

Comment:
We need to make this sentence bit simpler.

Reply:
Please do it for me. I too feel the same.

Reply:
Please check other localized versions to see how this was handled.

Reply:
`कंपनियों द्वारा कुबेरनेट्स को सफलतापूर्वक अपनाने में सहयोगी अनुभवी सेवा प्रदाता।` It is not that different from the old one, but still a bit easier to understand, what do you say @mittalyashu @Rajakavitha1 ?

Reply:
> `कंपनियों द्वारा कुबेरनेट्स को सफलतापूर्वक अपनाने में सहयोगी अनुभवी सेवा प्रदाता।`

1. Do we have to add `सफलतापूर्वक` in the above sentence?
2. In the English version, these are the following words which should be included in the Hindi version `service provider`, `deep experience`, `adopt Kubernetes`.

@saarques @Rajakavitha1